src/index.html

```diff
@@ -7,12 +7,58 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="imac image"
+        class="card__image"
+      />
+      <h3 class="card__name-product">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+      <p class="card__product-code">Product code: 195434</p>
+      <div class="reviews">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="reviews__title">Reviews: 5</p>
+      </div>
+      <div class="cost">
+        <p class="cost__name">Price:</p>
+        <p class="cost__value">$2,199</p>
+      </div>
+      <a
```

Comment:
Add the blank line between the blocks
```suggestion
      </div>
      
      <div class="cost">
        <p class="cost__name">Price:</p>
        <p class="cost__value">$2,199</p>
      </div>
      
      <a
```